backport-changelog/6.8/7069.md

```diff
@@ -2,3 +2,4 @@ https://github.com/WordPress/wordpress-develop/pull/7069
 
 * https://github.com/WordPress/gutenberg/pull/63401
 * https://github.com/WordPress/gutenberg/pull/66918
+* https://github.com/WordPress/gutenberg/pull/67018
```

Comment:
The plan is to backport all the block support stabilization updates in a single core PR as the PHP changes there will be done within relevant classes rather than via filters like the Gutenberg PRs. 

A single core commit will make debugging and tracing changes much easier as well.